openstack/glance/templates/glance-nanny-deployment.yaml

```diff
@@ -100,7 +94,7 @@ spec:
           readOnly: true
         {{- include "utils.proxysql.volume_mount" . | indent 8 }}
         {{- include "utils.trust_bundle.volume_mount" . | indent 8 }}
-      {{- tuple . (add .Values.rpc_workers .Values.workers) | include "utils.proxysql.container" | indent 6 }}
+      {{- tuple . (add .Values.glance_nanny.replicas) | include "utils.proxysql.container" | indent 6 }}
```

Comment:
No. Replicas will create more pods. Each pod has its own `proxysql` container and thus its own number of processes (one process, since there's only a single container with a single command). Using `replicas` here is not correct.

Reply:
value of .Values.glance_nanny.replicas is 1, we cannot leave it empty since the below is generated with `      {{- include "utils.proxysql.container" | indent 6 }}`

```
Error: Failed to render chart: exit status 1: Error: template: glance/templates/glance-nanny-deployment.yaml:97:10: executing "glance/templates/glance-nanny-deployment.yaml" at <include>: wrong number of args for include: want 2 got 1

Use --debug flag to render out invalid YAML

Error: plugin "diff" exited with error
```

Reply:
You can just put `1` in there though, right? If you ever increase the replicas, it will otherwise also increase the proxysql in every pod, too.